### PR TITLE
Hades: implement password reset link

### PR DESCRIPTION
**Pull request by akhilnarang**
Still needs to be tested and documented, and linked on the login page

### CODE REVIEW

File: hades/templates/reset_password.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <h1 class="h3 mb-3 font-weight-normal">Please enter your new desired password</h1>
+    <label for="inputUsername" class="sr-only">Username</label>
+    <input type="text" id="inputUsername" class="form-control" placeholder="Username" name="username"
+           required>
+    <br>
+    <label for="inputNewPassword" class="sr-only">New Password</label>
+    <input type="password" id="inputNewPassword" class="form-control" placeholder="New password" name="new_password"
+           required>
+    <!--
+    TODO: Add JS validation to ensure password has been entered again
```
Comment (prkprime) [inline hades/templates/reset_password.html:17]: Do you mean to check if old password is used or to check if another password has been inputted on the same page?

**Review by alr0cks — COMMENTED** [review]
It works!

File: hades/templates/reset_password.html (hunk 1 above)
Comment (akhilnarang) [reply hades/templates/reset_password.html:17]: I meant just to verify password, most change password pages make enter it twice for confirmation

**Review by prkprime — APPROVED** [review]
Everything seems fine. Also if anything breaks, #BlameAkeel xd

**Comment by akhilnarang** [comment]
All functionality seems fine, UI could probably be improved...

**Pull request merged**